torch/distributed/rpc/api.py

```diff
@@ -343,7 +346,7 @@ def shutdown(graceful=True, timeout=DEFAULT_SHUTDOWN_TIMEOUT):
             if not isinstance(agent, TensorPipeAgent) or agent.is_static_group:
                 _wait_all_workers(timeout)
                 _delete_all_user_and_unforked_owner_rrefs()
-                agent.join(shutdown=True)
+                agent.join(shutdown=True, timeout=timeout)
```

Comment:
lint error is real

Reply:
Updated `_distributed_rpc.pyi` to fix this